src/components/KToaster/ToastManager.cy.ts

```diff
@@ -0,0 +1,43 @@
+import ToastManager from './ToastManager'
+
+const toastersContainerId = 'kongponents-toaster-container'
+
+describe('ToastManager', () => {
+  it('should create a toasters container when initialized', () => {
+    const toastManager = new ToastManager()
+    const containers = document.querySelectorAll(`#${toastersContainerId}`)
+    expect(containers.length).equal(1)
+
+    // Clean up
+    toastManager.destroy()
+  })
+
+  it('should not create a duplicate toasters container if one already exists', () => {
+    const existingContainer = document.createElement('div')
+    existingContainer.id = toastersContainerId
+    document.body.appendChild(existingContainer)
```

Comment:
You should assert after this that this original container exists in the DOM before creating the new one